src/test/conftest.py

```diff
@@ -4,10 +4,30 @@
 # we'll get imports via path munging
 import sys
 import os
+
+import pytest_check as check
+
+import numpy as np
+
 sys.path.append(os.path.join(
     os.path.dirname(
         os.path.dirname(__file__)
     ),
     "main",
     "python"
-))
\ No newline at end of file
+))
+
+
+@check.check_func
```

Comment:
Why this way instead of `with check: assert np.allclose`? I don't have strong opinions but when you see these functions in the tests they don't immediately look like they are a check function

Reply:
fair. What if I called them e.g. `check_allclose`? That looks a bit like `check.equal` or `check.is_none` and seems to clearly delineate these as assertions.